File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    
    int i = 2;
    while(true)
    {
        if(n % i == 1)
            break;
        else
            ++i;
    }
    answer = i;
    
    return answer;
}

File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.cs
using System;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        
        int num = 2;
        while(true)
        {
            if(n % num == 1)
                break;
            
            ++num;
        }
        answer = num;
        
        return answer;
    }
}